itests/eth_fee_history_test.go

```diff
@@ -89,7 +76,15 @@ func TestEthFeeHistory(t *testing.T) {
 		}
 	}()
 
-	sort.Ints(tsHeights)
+	currTs, err := client.ChainHead(ctx)
+	require.NoError(err)
+
+	var tsHeights []int
+	for currTs.Height() != 0 {
+		tsHeights = append([]int{int(currTs.Height())}, tsHeights...)
```

Comment:
Not sure of the intent here, but if if you want reverse order, would it not be better/readable to call append like normally and then sort afterwards?